lib/core/enhanceError.js

```diff
@@ -16,12 +16,12 @@ module.exports = function enhanceError(error, config, code, request, response) {
     error.code = code;
   }
 
-  error.request = request;
+  error.request=request;
```

Comment:
format: style mismatch:
```suggestion
error.request = request;
```

`2f4f64f1` ⏎⏎ at column 4 should be removed.
`4e4e06c5` ␣ should be inserted at column 16.
`a04c23e2` ␣ should be inserted at column 17.